### ISSUE TITLE
Wrong pot for fusion recipe and some small issues

### ISSUE THREAD

**Issue by firecat6666** [Ruin0x11/ElonaPlusCustom_GX#32]
The very first one, Create Junk, requires 4 items instead of 5. This was changed way back in 1.78 so I'm here wondering what other changes and additions might be missing from Custom...

The relevant change quoted from the english changelog:
"The ingredients for the junk make recipe no longer accept items that you can grab an infinite amount of. Also increased the number of items needed by 1 so that you can't keep using the recipe by inputting products as ingredients."

Also, there's a small cosmetic issue with the journal. In the section "Records of Adventure" that appears when the "Record" option under Game Settings is set to Show Max, there's some extra empty lines or something. This causes some lines to spill over to the following pages, with two of them spilling over to page 9 when the game is in english and one spilling over when it's in japanese. In the non-Custom game, they all fit neatly in page 8, as per the screenshot, and page 9 is empty (although only in japanese, that journal section doesn't appear in the english non-Custom).

![sadsadas](https://user-images.githubusercontent.com/94730925/142715664-3bb9ed26-4c43-4dbf-a7fb-dfb686c85591.png)

**Comment by firecat6666** [comment]
The following recipes are also wrong, although I think these were changed intentionally for some reason:

Ball making: also requires an api nut in addition to the original ingredients (pokemon reference?).
Ground Bait: also requires one of any food in addition to the original ingredients.
Barrel bomb: completely changed recipe except for the barrel (this new recipe makes more sense than the original). Also, the recipe says "Add any barrel", would have been fine if it was just "Add barrel".

Some other text/translation issues:

The recipe for "Crystal bomb" should have been called Magistus, as per the item's actual name.
Same with "Big crystal bomb" (Trismagistus). Should also change its requirements to say "Add magistus".
Same issue as the above with the next two recipes, Evil sword/True evil sword (actual names: Skull sword/Violent skull sword)
The equip making recipes say "Add stardust, etc" ("Add stardust etc", without the comma, without Custom), but it's just a stardust (no etc required).
Moved from the other issue: Also, there's a small translation issue with the recipe called Harmful potion. In Custom it's translated as Antidote, which is completely innacurate considering what it creates. I think Harmful potion is good enough.

**Comment by firecat6666** [comment]
I found a couple more inconsistencies between Custom and vanilla:

This change from 1.995: "Clarified the UI for inventory mode swapping." wasn't applied in Custom (at least I think that's the relevant change, maybe it was something else in a different version). It makes the "7 ,9 ,Tab [Change]" line that appears when you open an inventory window, under the row of icons on the top right, RED AND BOLD and centered, and I think the font is slightly bigger too. This was only changed for inventory menus (use, drop, read, etc), not character menus (chara, wear, etc). Also, in that line, the spaces appear before the commas if you want to fix that, but that's wrong in the japanese version too...

In Custom, when you place a deed over a road tile, the road disappears permanently, like how forests and other decorated wilderness tiles disappear when you place a deed or a dungeon spawns on them, causing slow, inexorable deforestation... I tested the road disappearing thing in Custom 1.76fix and it doesn't happen there, or in vanilla.

Truck isn't doing the honk sound when pushing 's' in the world map. The sound on all other 3 vehicles is working correctly, only truck isn't for whatever reason. All 4 work fine in vanilla.

**Comment by firecat6666** [comment]
There's a line in the journal statistics that's mistranslated. In the image in the above post, it's the last blue line in the middle page. In japanese it reads "spell bonus chineseletterswhatdotheymean", and it keeps track of total spell bonus points earned. In Custom/english it's translated as "Total Skill Points Earned".

Additionally, Custom isn't keeping track of skill bonus points earned, so the line above that one shows a wrong value (0 in my case). I tested this by opening my save, which I've only played in Custom 2.07, in a vanilla game, gained some levels and the number increased from 0. Then I saved, went back to Custom, gained levels again but the number didn't increase.

**Comment by firecat6666** [comment]
Another old change that isn't implemented in this version of Custom, from 1.76:

"Corrected the danger level displayed for Fort of Chaos Nefia. Their actual danger level remains the same. It used to be more for flavor than to reflect the actual difficulty of what's inside, but players had to look things up on the internet to be able to know if they're ready for it."

This change WAS implemented in Custom 1.76fix, though, so I don't know what went wrong/is missing/was forgotten between Custom versions. I've attached the aforementioned 1.76 version if you want to compare the code or something.
[ElonaCustom 1.76fix.1.zip](https://github.com/Ruin0x11/ElonaPlusCustom-GX/files/7614700/ElonaCustom.1.76fix.1.zip)

**Comment by firecat6666** [comment]
Regarding the second sentence in this change from 1.94:

"Made ranches start with a big brush on the ground. You really need only one, so big brushes no longer spawn randomly."

I don't know if it was implemented in properly in vanilla, and I don't want to try testing it, but I've found a couple big brushes on the floor of random dungeons in Custom. Ranches are starting with a big brush on the ground, though.
